### DISCUSSION TITLE
[Locked] Troubleshooting Failed Installation

### DISCUSSION THREAD

**Discussion by Chireven** [actions/virtual-environments#2310] [Image configuration]
Greetings:

I am attempting to use this project to build on-premise devops build agents.  I have modified the builder to build the VM in hyper-v, but I'm struggling with the software install.

After two hours or building, it fails with :
    hyperv-iso: Starting Install Azure Dev Spaces CLI.exe...
    hyperv-iso: Non zero exit code returned by the installation process: -2146233318

What is the best way to troubleshoot this?  

Any help would be greatly appreciated!!

**Comment by AlenaSviridenko** [comment]
Hi @Chireven, for things that are not working on the image or during image generation, please, file an issue and our engineers will investigate it.
